### DISCUSSION TITLE
Text popup blocks input

### DISCUSSION THREAD

**Discussion by Quimap** [AppGround-io/bluetooth-keyboard-and-mouse-support#169] [Bugs & Problems]
The text popup blocks sometimes mouse and keyboard inputs.

Example:
![screen](https://user-images.githubusercontent.com/87367117/126244647-5c2fe866-c293-450c-8a1e-9bdc9fa2b0ff.jpg)

**Comment by zorazorr** [comment]
Fixed: Version 4.3.0

Thank you!
